lib/Console/Command/Parse.php

```diff
@@ -288,7 +297,6 @@ protected function populateStatuses(\SimpleXMLElement $xmlStatuses) {
 
     protected function generateCommitsLog(\SimpleXMLElement $xml, array $commits) {
         foreach (array_reverse($commits) as $commit) {
-            \Mesamatrix::$logger->debug('Processing commit '.$commit->getHash());
```

Comment:
I removed that because it's not really needed anymore since it's just basic XML processing, nothing wrong can happen.